node/tests/GlideClient.test.ts

```diff
@@ -203,6 +200,29 @@ describe("GlideClient", () => {
         },
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
We need the same for default string.
Do we test both default on transaction as well?

Reply:
the default is string so all the tests already runs with string decoder. but added